requirements.txt

```diff
@@ -1,11 +1,11 @@
 # The "base" version of the Mkdocs project.
 # Note: if you update this, also update `install_requires` in setup.py
```

Comment:
Seems this comment should have been removed with 9b7a77e7bd18a321c8fe614ab516c691c7537740